src/test/kotlin/com/franzi/todochallenge/controller/TaskRepositoryTest.kt

```diff
@@ -0,0 +1,44 @@
+package com.franzi.todochallenge.controller
+
+import com.franzi.todochallenge.repository.TaskRepository
+import org.junit.jupiter.api.Test
+
+import org.assertj.core.api.Assertions
+import java.time.Instant
+
+internal class TaskRepositoryTest {
+
+    private val taskRepository = TaskRepository()
+
+    @Test
+    fun insertAndFind() {
```

Comment:
Naming tests like "Should do something when some precondition is given" is a good practice we could leverage here. TBH, I don't see any preconditions for this case, so, the following should do

```suggestion
    fun`Should include Tasks properly`() {
```

Reply:
done also changed the other names in this file